src/antq/diff/github_tag.clj

```diff
@@ -18,4 +20,5 @@
                     (:version dep))
         latest (or (exact-or-included tags (:latest-version dep))
                    (:latest-version dep))]
-    (format "%s/compare/%s...%s" url current latest)))
+    (when (and current latest)
+      (format "%s/compare/%s...%s" url current latest))))
```

Comment:
I'm concerned about the fact that you are doing nil checks twice.
For example, what do you think doing nil check once at the beginning as follows?

```clojure
(defmethod diff/get-diff-url :github-tag
  [{:as dep :keys [version latest-version]}]
  (when (and version latest-version) ;; <- HERE
    (let [url (format "https://github.com/%s"
                      (str/join "/" (take 2 (str/split (:name dep) #"/"))))
          tags (u.git/tags-by-ls-remote url)
          current (or (exact-or-included tags version)
                      version)
          latest (or (exact-or-included tags latest-version)
                     latest-version)]
      (format "%s/compare/%s...%s" url current latest))))
```

Reply:
Thank you for the review and all the suggestions. I've changed it to this.